node/constants.ts

```diff
@@ -55,7 +55,6 @@ export const {
   SIGBUS,
   SIGCHLD,
   SIGCONT,
-  SIGEMT,
```

Comment:
Sadly since this has to be a named export, all the constants have to be explicitly named. There are probably quite a few that weren't included here because of that (or even this one that I couln't find anywhere)